met-web/src/components/engagement/admin/create/authoring/AuthoringTemplate.tsx

```diff
@@ -108,7 +109,8 @@ const AuthoringTemplate = () => {
             <SystemMessage status="warning">
                 Under construction - the settings in this section have no effect.
             </SystemMessage>
-            {'details' === slug && (
+
+            <When condition={'details' === slug}>
```

Comment:
Oh yes, this was the only modification to the Details section. I wanted to make it consistent with the rest of our code, where we use <When> often instead of the && conditional.